ReactAndroid/src/main/java/com/facebook/react/ReactInstanceManagerBuilder.java

```diff
@@ -66,6 +66,17 @@ public class ReactInstanceManagerBuilder {
   private @Nullable Map<String, RequestHandler> mCustomPackagerCommandHandlers;
   private @Nullable ReactPackageTurboModuleManagerDelegate.Builder mTMMDelegateBuilder;
   private @Nullable SurfaceDelegateFactory mSurfaceDelegateFactory;
+  /**
+   * An enum that specifies the JS Engine to be used in the app
+   * Old Logic uses the legacy code
+   * JSC/HERMES loads the respective engine using the revamped logic
+   */
+  public enum JSInterpreter {
+    OLD_LOGIC,
+    JSC,
+    HERMES
+  }
```

Comment:
Let's keep this in a separate file as it was before

Reply:
Android CI tests are failing in that case, I tried 3 times.

Reply:
Tests were failing as you used a `import com.facebook.react.util.JSInterpreter;`. You should be good placing `JSInterpreter` inside the `com.facebook.react` package

Reply:
done @cortinico please review